fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -246,6 +246,7 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
 
     public static final String RUNTIME_FILTER_SCAN_WAIT_TIME = "runtime_filter_scan_wait_time";
     public static final String RUNTIME_FILTER_ON_EXCHANGE_NODE = "runtime_filter_on_exchange_node";
+    public static final String ENABLE_MULTI_COLUMNS_ON_GLOBAL_RUNTIME_FILTER = "multi_columns_on_global_runtime_filter";
```

Comment:
suggest to use "enable_multicolumn_global_runtime_filter" instead of "multi_columns_on_global_runtime_filter"